task/output.go

```diff
@@ -54,20 +61,43 @@ func (t *Output) Flush() {
 }
 
 // InitBar is needed for progress compatibility
-func (t *Output) InitBar(count int64, isBytes bool) {
+func (t *Output) InitBar(count int64, isBytes bool, name string) {
 	// Not implemented
 }
 
+// InitBar publish output specific
+func (t *PublishOutput) InitBar(count int64, isBytes bool, name string) {
+	t.bar = name
+	if name == "generate-package-files" {
```

Comment:
Couldn't we do constants for those progressbar names? Otherwise we may not catch typos. Then use only those constants.. Especially since we actually influence control flow in this case.

Reply:
Fair point. I have replaced the strings with bar type constants.